content/docs/started/getting-started-k8s.md

```diff
@@ -144,11 +144,342 @@ kubectl get cm dex -n kubeflow -o jsonpath='{.data.config\.yaml}' > dex-config.y
+                      value: "Arrikto"
+                    - name: LDAP_DOMAIN
+                      value: "arrikto.com"
+                    - name: LDAP_ADMIN_PASSWORD
+                      value: "admin"
+                    - name: LDAP_CONFIG_PASSWORD
+                      value: "config"
+                    - name: LDAP_BACKEND
+                      value: "mdb"
+                    - name: LDAP_TLS
+                      value: "false"
+                    - name: LDAP_REPLICATION
+                      value: "false"
+                    - name: KEEP_EXISTING_CONFIG
+                      value: "false"
+                    - name: LDAP_REMOVE_CONFIG_AFTER_SETUP
+                      value: "true"
+              volumes:
+                - name: ldap-config
+                  emptyDir: {}
+          volumeClaimTemplates:
+              - metadata:
+                  name: ldap-data
+                spec:
+                  accessModes: [ "ReadWriteOnce" ]
+                  resources:
+                    requests:
+                      storage: 10Gi
+        
+        ---
+        
+        apiVersion: v1
+        kind: Service
+        metadata:
+          labels:
+            app: phpldapadmin
+          name: phpldapadmin-service
+          namespace: kubeflow
+        spec:
+          type: ClusterIP
+          ports:
+            - port: 80
+          selector:
+            app: phpldapadmin
+        
+        ---
+        
+        apiVersion: apps/v1
+        kind: Deployment
+        metadata:
+          name: phpldapadmin
+          namespace: kubeflow
+          labels:
+            app: phpldapadmin
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              app: phpldapadmin
+          template:
+            metadata:
+              labels:
+                app: phpldapadmin
+            spec:
+              containers:
+                - name: phpldapadmin
+                  image: osixia/phpldapadmin:0.8.0
+                  ports:
+                    - name: http-server
+                      containerPort: 80
+                  env:
+                    - name: PHPLDAPADMIN_HTTPS
+                      value: "false"
+                    - name: PHPLDAPADMIN_LDAP_HOSTS
+                      value : "#PYTHON2BASH:[{'ldap-service.kubeflow.svc.cluster.local': [{'server': [{'tls': False}]},{'login': [        {'bind_id': 'cn=admin,dc=arrikto,dc=com'}]}]}]"
+        {{< /highlight >}}
+        
+        </details>
+        
+        
+    2. Seed the LDAP database with new entries.
+        
+        ```bash
+        kubectl exec -it -n kubeflow ldap-0 -- bash
+        ldapadd -x -D "cn=admin,dc=arrikto,dc=com" -W
+        # Enter password "admin".
+        # Press Ctrl+D to complete after pasting the snippet below.
+        ```
+        
+        <details>
+        
+        <summary>LDAP Seed Users and Groups</summary>
+        ```ldif
+        # This already exists.
```

Comment:
I think the comment needs a little more explanation:

```
        # This already exists.
        # If it doesn't, uncomment this.
```

For example, what already exists (expand on "this"), how do people know it already exists?

Reply:
This already exists because it is created by the OpenLDAP image we use, based on the parameters we have give it.
Will elaborate to make it more clear.